docs/user-guide/rules.md

```diff
@@ -209,16 +209,16 @@ Here are all the rules within stylelint, grouped first [by category](../../VISIO
 
 #### Number
 
--   [`number-leading-zero`](../../lib/rules/number-leading-zero/README.md): Require or disallow a leading zero for fractional numbers less than 1.
--   [`number-no-trailing-zeros`](../../lib/rules/number-no-trailing-zeros/README.md): Disallow trailing zeros in numbers.
+-   [`number-leading-zero`](../../lib/rules/number-leading-zero/README.md): Require or disallow a leading zero for fractional numbers less than 1 (Autofixable).
+-   [`number-no-trailing-zeros`](../../lib/rules/number-no-trailing-zeros/README.md): Disallow trailing zeros in numbers (Autofixable).
 
 #### String
 
 -   [`string-quotes`](../../lib/rules/string-quotes/README.md): Specify single or double quotes around strings.
 
 #### Length
 
--   [`length-zero-no-unit`](../../lib/rules/length-zero-no-unit/README.md): Disallow units for zero lengths.
+-   [`length-zero-no-unit`](../../lib/rules/length-zero-no-unit/README.md): Disallow units for zero lengths (Autofixable).
```

Comment:
Not related to this PR. Addressed in https://github.com/stylelint/stylelint/pull/2957.